.travis.yml

```diff
@@ -2,14 +2,14 @@ sudo: false
 language: node_js
 cache:
   directories:
+    - $HOME/.npm
     - node_modules
 node_js:
   - "4"
   - "6"
+  - "8"
 before_install:
-  # remove outdated deps, assists with cache maintenance
```

Comment:
With npm v5.x cache has been rewritten and this type of cache purging is no longer required

There's a good 10 or so mentions of the vast and fantastic cache improvements mentioned on these two posts that I think is worth a quick read, hit CTRL+F and search for _cache_ on these two posts:
http://blog.npmjs.org/post/161276872334/npm5-is-now-npmlatest
http://blog.npmjs.org/post/161081169345/v500

😄